components/Bounty/api/fetchBountiesAPI.ts

```diff
@@ -1,17 +1,20 @@
-import API from "~/config/api";
-import { buildApiUri } from "~/config/utils/buildApiUri";
+import API, { generateApiUrl } from "~/config/api";
 import { Helpers } from "@quantfive/js-web-config";
+import { PaginatedApiResponse } from "~/config/types/root_types";
 
 type Args = {
-  onError: (error: Error) => void;
-  onSuccess: (response: any) => void;
+  personalized?: boolean;
   page?: number;
 };
 
-export const fetchBounties = ({ onError, onSuccess, page }: Args): void => {
-  fetch(buildApiUri({ apiPath: "bounty/get_bounties" }), API.GET_CONFIG())
+export const fetchBounties = ({ personalized = true, page }: Args): Promise<PaginatedApiResponse> => {
```

Comment:
Will changing this also affect the _Open Bounties_ side bar that uses a non-paged format and fetched a maximum of 10?